src/pytest_plugins/consume/releases.py

```diff
@@ -72,7 +73,11 @@ def __eq__(self, value) -> bool:
         """
         assert isinstance(value, str), f"Expected a string, but got: {value}"
         if self.version is not None:
-            return value == f"{self.tag_name}@{self.version}"
+            # normal release, e.g., stable@v4.0.0
+            normal_release_match = value == self.version
```

Comment:
This is a bit of a quick fix. I think this will break once a pre-release version matches a regular release version (although quite unlikely).

We need to better abstract this logic as `pectra-devnet-6@v1.0.0` is a valid tag for a pre-release, but `v4.2.0` is a valid tag for a regular release.